ch8/tiling2.py

```diff
@@ -0,0 +1,18 @@
+MOD = 1000000007
+def tiling(width):
+	if width <= 1:
+		return 1
+
+	if width in cache:
+		return cache[width]
+
+	ret = (tiling(width - 2) + tiling(width - 1)) % MOD
+	cache[width] = ret
+	return ret
+
+
+C = input()
+for _ in xrange(C):
+	n = input()
+	cache = {}
+	print tiling(n) % MOD
```

Comment:
Question: 
Is "% MOD" after "tiling(n)" necessary?
Isn't the answer to tiling(n) already taken account for "% MOD" in line 9?

Reply:
good eye. not necessary.